apps/happy-blocks/src/pricing-plans/components/block-settings.tsx

```diff
@@ -0,0 +1,36 @@
+import { InspectorControls } from '@wordpress/block-editor';
+import { BlockEditProps } from '@wordpress/blocks';
+import { SelectControl, PanelRow, PanelBody } from '@wordpress/components';
+import { FunctionComponent } from 'react';
+import { BlockAttributes, PricingPlansConfiguration } from '../types';
+
+interface Props {
+	plans: PricingPlansConfiguration;
+}
+
+const BlockSettings: FunctionComponent<
+	Pick< BlockEditProps< BlockAttributes >, 'attributes' | 'setAttributes' > & Props
+> = ( { attributes, setAttributes, plans } ) => {
+	const options = Object.values( plans ).map( ( plan ) => ( {
+		value: plan.billing.slice( -1 )[ 0 ].planSlug,
+		label: plan.label,
+	} ) );
+
+	return (
+		<InspectorControls>
+			<PanelBody title="Basic" initialOpen={ true }>
+				<PanelRow>
+					<SelectControl
+						className="wp-block-a8c-pricing-plans__settings-plan"
+						label="Plan"
```

Comment:
Should we add translation strings instead of hardcoded ones here and in other parts of the PR?